### ISSUE TITLE
How is it supposed to work?

### ISSUE THREAD

**Issue by kellertobias** [scr34m/Signature#1]
I have done all steps in the Readme (Set the flag to EnableBundles, Install the latest build then Allow Signature.mailbundle in Manage Plug-ins). afterwards, mail restarted.

However I don't find any new menu items. How do I setup which signature belongs to which outbound address?

Mail Version 16.0, macOS Ventura 13.3.1b

**Comment by scr34m** [comment]
In the settings menu under signatures tab you have to add as many e-mail address as you want, and if you reply an e-mail then it will automatically select the correct signature.

And under mail boxes tab in the e-mail dropdown set sender addresses you want to use.

**Comment by kellertobias** [comment]
I have created 4 Signatures for the Account and named the signatures like the outbound addresses.
![Screenshot 24 05 2023 um 14 35 53 PM](https://github.com/scr34m/Signature/assets/6940430/c422290b-870b-463d-a71c-8ad85a172a8a)

If I now select the mail address when composing an email, it always jumps back to the first Signature I created.

![Screenshot 24 05 2023 um 14 38 18 PM](https://github.com/scr34m/Signature/assets/6940430/a6d271a9-4c06-40b1-b982-9b7a30afe81e)

As reference:

![Screenshot 24 05 2023 um 14 39 27 PM](https://github.com/scr34m/Signature/assets/6940430/dbee3b6f-c381-4ca9-9bcf-93deab49a7a8)
![Screenshot 24 05 2023 um 14 39 43 PM](https://github.com/scr34m/Signature/assets/6940430/21bfc67d-10f7-4e3c-bc4e-4e8299be5496)

**Comment by scr34m** [comment]
Then if you change the from address it will automatically change the signature in the compose window.
And so if you reply automatically select the correct signature.

This plugin will not shown up under tha plugins tab, because this is the old ways to develop plugins.

**Comment by kellertobias** [comment]
For answering it seems to work, but for new emails, it seems that the selection of the Signature needs to be set to "Random", otherwise it flickers (quickly show the actually correct signature, then flickers back to the old one)

**Comment by scr34m** [comment]
That is strange, i use daily and never noticed this behaviour, any other plugin installed?

**Comment by kellertobias** [comment]
Nope. Just started using apple mail. Had used Mimestream earlier until I now got greeted with "Hey We updated to version 1.0 Please pay now or you cannot use it"
